nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the install medium {#sec-installation-booting}
+
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Plug in the install drive. Then turn on or restart your computer.
+
+2.   Press <kbd>F12</kbd> (or the appropriate key) to open the boot menu.
+     Select the USB flash drive (the option usually contains the word "USB").
+     If you choose the incorrect drive, your computer will likely continue to
+     boot as normal. In that case restart your compute and pick a different drive.
+
+     ::: {.note}
+     The key to open the boot menu is different across computer brands and even
+     models. It is usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd> or
+     another function key. If you are unsure, look at the POST screen that is
```

Comment:
```suggestion
     models. It can be <kbd>F12</kbd>, but also <kbd>Esc</kbd>, <kbd>Enter</kbd> or
     another function key. If you are unsure, look at the POST screen that is
```

Reply:
thanks. i also added all keys you mentioned above